ykcapi/src/lib.rs

```diff
@@ -278,7 +329,8 @@ pub extern "C" fn __llvm_deoptimize(
     // stackmap).
     unsafe {
         asm!(
-            // Save registers to the stack.
+            // Save registers the may be referenced by the stackmap to the stack before they get
```

Comment:
Sentence doesn't parse. I'm not sure if it's supposed to be "THAT may be referenced..."?

Reply:
Sorry, yes. It's supposed to be `Save registers that`. Will fix.

Reply:
Fixed in [c45e5e6](https://github.com/ykjit/yk/pull/578/commits/c45e5e644b2bc6b43c9d456a8192a5cc3ddbaff8)